requirements.txt

```diff
@@ -9,3 +9,4 @@ sentry-sdk~=1.5
 flake8
 pylint
 wheel
+nose
```

Comment:
nose is probably discontinued, so it will be a good idea to move to pytest:
https://nose.readthedocs.io/en/latest/#note-to-users